idunn/directions/client.py

```diff
@@ -77,15 +126,20 @@ def directions_combigo(self, start, end, mode, lang, **kwargs):
             data=response.json()
         ).dict()
 
+
     def get_directions(self, from_loc, to_loc, mode, lang, **extra):
+        default_method = self.directions_qwant
+        if self.MAPBOX_API_ENABLED:
+            default_method = self.directions_mapbox
+
         if mode in ('driving-traffic', 'driving', 'car'):
-            method = self.directions_qwant
+            method = default_method
```

Comment:
Instead of repeating this line in all `if` blocks, why not setting it once before all `if else`?

Reply:
done